api/src/index.ts

```diff
@@ -7,26 +7,70 @@ import { authenticationRouter, verificationRouter, channelInfoRouter, channelRou
 import { MongoDbService } from './services/mongodb-service';
 import { CONFIG } from './config';
 import * as expressWinston from 'express-winston';
-import { setupApi } from './setup';
+// import { checkRootIdentity, setupApi } from './setup';
 import swaggerJsdoc from 'swagger-jsdoc';
 import { Logger } from './utils/logger';
 import { openApiDefinition } from './routers/swagger';
 import { serverInfoRouter } from './routers/server-info';
+import yargs from 'yargs';
+import { KeyGenerator } from './setup';
+import { KeyResolver } from './setup/key-resolver';
 
 const logger = Logger.getInstance();
 
+const argv = yargs
+    .command('server', 'Start the integration service API', {})
+    .command('keygen', 'Generate root identity for integration service API', {})
+    .help()
+    .alias('help', 'h')
+    .argv;
+
 function useRouter(app: express.Express, prefix: string, router: express.Router) {
 	const messages = router.stack.map((r) => `${Object.keys(r?.route?.methods)?.[0].toUpperCase()}  ${prefix}${r?.route?.path}`);
 	messages.map((m) => logger.log(m));
 
 	app.use(prefix, router);
 }
 
+async function hasRootIdentity(): Promise<string> {
```

Comment:
from this name I would expect to receive a boolean if there is an root identity or not
=> imo `getRootIdentity` or `getRootIdentityId` would be the better name since you actually return the identity id